### ISSUE TITLE
Unable to upload SVG images for categories

### ISSUE THREAD

**Issue by mudgie** [opencart/opencart#3064]
Using OpenCart 2.0.1.1, I receive the following error when attempting to upload an SVG image as a product category image:
![2015-05-30 19_47_33](https://cloud.githubusercontent.com/assets/1667777/7896688/36a2b3c6-0705-11e5-80c8-cd1b063a43ae.png)

SVGs are allowed in 'Allowed File Extensions' and 'Allowed File Mime Types' within the Server options (I have not modified this from the default installation):
![2015-05-30 19_52_14](https://cloud.githubusercontent.com/assets/1667777/7896696/97bacc2a-0705-11e5-910b-c09705a00b13.png)

I've double checked that the MIME type of my SVG image is image/svg+xml.

**Comment by GomDani** [comment]
Look in filemanager.php at line: 234 and 245

Settings has nothing to do with the upload in filemanager.

**Comment by mudgie** [comment]
I did look into this a bit further and did find those lines in filemanager.php. I've added 'svg' and 'image/svg+xml' to the $allowed array, however, it goes a bit deeper than that. Doing this allowed the uploading of SVGs however the images don't display in the admin panel properly.

I assume that the image resizing is failing because it's not a file type that OpenCart supports properly, so to work around the issue I'm modifying template files such as \admin\view\template\common\filemanager.tpl and \admin\view\template\catalog\category_form.tpl with code like:

<?php
	if (preg_match("/\.svg/",$thumb)) {
		$thumb = str_replace("-100x100","",$thumb);
		$thumb = str_replace("/cache","",$thumb);
	}
?>

As it doesn't copy the SVG to /image/cache/catalog/ with '-100x100' in the file name, I'm simply removing those parts so the image path is legitimate. 

I know that this isn't the most ideal way to work around the issue, however I'm not very familiar with how OpenCart is put together.
